mailpoet/assets/js/src/segments/dynamic/templates/components/template_list_item.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  Tooltip,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { starFilled } from '@wordpress/icons';
+import { Tag } from '@woocommerce/components';
+
+type TemplateListItemProps = {
+  template: {
+    id: number;
```

Comment:
Is the ID being used for anything?

Reply:
It was not being used. Now it is being used for the `key` parameter that is passed to `<TemplateListItem>`. Since the hard coded templates will be replaced in a subsequent ticket, I haven't put much thought into the data structure of the templates in this ticket.

Reply:
Just wanted to make sure I wasn't missing anything, thanks!